File: BonjourApi/BonjourApi/Models/FriendshipRelation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BonjourApi.Models
{
    public class FriendshipRelation
    {
        public int Id { get; set; }
        public int firstUserId { get; set; }
        public int secondUserId { get; set; }
        public Boolean isFriend { get; set; }

    }
}

File: android-app/bonjour2/app/src/main/java/com/example/bonjour/FriendshipRelation.java
package com.example.bonjour;

/**
 * Created by FATEHULLAH on 4/6/2015.
 */
public class FriendshipRelation {
    public int Id ;
    public int firstUserId ;
    public int secondUserId ;
    public Boolean isFriend ;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public int getFirstUserId() {
        return firstUserId;
    }

    public void setFirstUserId(int firstUserId) {
        this.firstUserId = firstUserId;
    }

    public int getSecondUserId() {
        return secondUserId;
    }

    public void setSecondUserId(int secondUserId) {
        this.secondUserId = secondUserId;
    }

    public Boolean getIsFriend() {
        return isFriend;
    }

    public void setIsFriend(Boolean isFriend) {
        this.isFriend = isFriend;
    }
}
